huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_cassandra_instance.go

```diff
@@ -122,7 +129,7 @@ func ResourceGeminiDBInstanceV3() *schema.Resource {
 							Required: true,
 							ForceNew: true,
 							ValidateFunc: validation.StringInSlice([]string{
-								"GeminiDB-Cassandra",
+								"cassandra",
```

Comment:
docs also should be updated

Reply:
use "GeminiDB-Cassandra" will cause ForceNew, so we should remove it from the doc